generators/app/utils/messages.js

```diff
@@ -1,21 +1,29 @@
 "use strict";
 const chalk = require("chalk");
 
-function mainMessage(text) {
-  return `${chalk.underline(text)}`;
+function message(text, lineBreak = false, indentation = 0, icon = "") {
+  const lineBreakString = lineBreak ? "\n" : "";
+  const indentationString = " ".repeat(indentation);
+  const iconString = icon ? `${icon} ` : "";
+  return `${lineBreakString}${indentationString}${iconString}${chalk.reset(
+    text
```

Comment:
What is the purpose of the `chalk.reset`? Do you any text style going through this function? 
Maybe a test could be a nice documentation!

Reply:
Yes, it's possible to add some style without removing it, and Yeoman does that, which lead to different colors for messages printed with the same util, that's why I've added that.
For the test, I plan to write them in the future, there is a ticket too :)